tensorflow-core/tensorflow-core-api/src/gen/annotations/org/tensorflow/op/Ops.java

```diff
@@ -7726,7 +7740,7 @@ public Ops withName(String opName) {
   }
 
   /**
-   * Returns an API that uses the provided DeviceSpec for an op.
+   * Returns an API that places the created operations on the device(s) matching the provided spec.
```

Comment:
Ah, looks like something was out of sync with the last PR that was merged, good thing that we'll fix it here.